tools/scripts/generate_changelog.py

```diff
@@ -0,0 +1,62 @@
+"""
+Generate changelog for github release from the changelog.rst file
+Example of execution:
+
+python tools/scripts/generate_changelog.py -v 1.15.0 -f docs/changelog.rst
+
+It outputs the result to the stdout
+"""
+
+import argparse
+import re
+
+
+def generate_changelog(version: str, path: str) -> str:
+    change_regex = r'\*\ :(feature|bug):`(\d*|-)`\ ([\w| |.]+)'
+
+    changes = {'features': [], 'bugs': []}
+    github_link = '- [{number}](https://github.com/rotki/rotki/issues/{number}) {text}'
```

Comment:
Having the entire link is not needed. Simply write the issue number.

As you can see this is how we have been doing it:

![2021-05-04_16-28_1](https://user-images.githubusercontent.com/1658405/117019523-c2cb4b00-acf5-11eb-8514-b27e340475ba.png)
![2021-05-04_16-28](https://user-images.githubusercontent.com/1658405/117019530-c4950e80-acf5-11eb-8224-ce4c8add436f.png)

It's not wrong of course! But using the number with the `#` will be transferrale to any repo, even if we rename the repo. Using the full link will not be.

On the other hand using the full link would work if we ever split into smaller repos. I am not 100% sure but for now keeping it as it was is probably the best idea.